.github/workflows/mobile-deploy-auto.yml

```
@@ -22,21 +22,21 @@ jobs:
       platforms: ${{ steps.check.outputs.platforms }}
-    
+
     steps:
       - name: Check deployment conditions
         id: check
         run: |
           echo "🔍 Checking deployment conditions..."
-          
+
           # Skip if PR has skip-deploy in title or body
           if [[ "${{ github.event.pull_request.title }}" =~ \[skip-deploy\] ]] || 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)